### ISSUE TITLE
[General]: Define interface to Intermediate Layer

### ISSUE THREAD

**Issue by Vroni27** [una-auxme/paf#554]
### Detailed Description

We should provide a list of requirements we need from the intermediate layer.
To add requirements, write a comment under this issue.

### Dependencies

- #578

**Comment by Vroni27** [comment]
For the ACC:

- [ ] Distance to object driving in front (car, biker, etc.) is needed
- [ ] Velocity of the object in front is needed

For lane change, intersection:

- [ ] Is the left lane free?
- [ ] Is there any oncoming traffic?

**Comment by Vroni27** [comment]
To be defined: Interface (Information published on various topics, service-client-communication, ...?)

**Comment by asamluka** [comment]
Shouldn't the lane change check the right lane too, to
1. check if a merge back to the original lane is possible
2. ensure, that no collision with a vehicle to right is happening while merging back
3. make lane changing more universal

**Comment by Zelberor** [comment]
Can this be closed?

**Comment by Vroni27** [comment]
Yes, this can be closed since the intermediate layer is already integrated.
